src/sentry/api/urls.py

```diff
@@ -2741,8 +2742,15 @@ def create_group_urls(name_prefix: str) -> list[URLPattern | URLResolver]:
         SentryInternalAppTokensEndpoint.as_view(),
         name="sentry-api-0-sentry-internal-app-tokens",
     ),
+    # The order of the next 2 urls matters. We want to let the numeric id handler try to match the regex first,
+    # before moving to alphanumeric which is a catch-all.
     re_path(
-        r"^(?P<sentry_app_slug>[^\/]+)/api-tokens/(?P<api_token>[^\/]+)/$",
+        r"^(?P<sentry_app_slug>[^\/]+)/api-tokens/(?P<api_token>\d+)/$",
```

Comment:
🤔 
actually - _does_ the route param here need to be the same?
could you name this `/(?P<token_id>\d+)/` here and if the regex doesn't match, it'll still fall through?

Reply:
No, we can't do this right now, but it will get updated very shortly. I just need this PR as an intermediary for the transition to get FE changed safely.

Reply:
The name here needs to be the same as the name in the `convert_args` method, so we're limited here unfortunately

Reply:
And we can't change that because of the python child/parent overloading haha. It's all intentional changes, but merely temporary until the next PR